Makefile

```
@@ -1,16 +1,21 @@
+UNAME_S := $(shell uname -s)
 MAKEPATH:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 NAMESPACE:=rancher-logging-example
 NAME:=$(NAMESPACE)
 
-install: uninstall-example
+install:
 	helm install -n $(NAMESPACE) --create-namespace $(NAME) $(MAKEPATH)/charts/rancher-logging-example
 
 uninstall:
-	-helm uninstall -n $(NAMESPACE) $(NAME)
-	-kubectl delete namespace $(NAMESPACE)
+	- helm uninstall -n $(NAMESPACE) $(NAME)
+	- kubectl delete namespace $(NAMESPACE)
```

What does this change do? We use the `-$COMMAND` notation to ignore errors. This is done since uninstallation should be allowed to fail if the chart is already uninstalled.